src/pages/case-study/central-working-changing-how-you-work.js

```diff
@@ -0,0 +1,385 @@
+import React, { Fragment } from 'react'
+import { graphql } from 'gatsby'
+import styled from 'styled-components'
+import breakpoint from 'styled-components-breakpoint'
+import ReactMarkdown from 'react-markdown'
+import { generate } from 'shortid'
+
+import { Grid, Col } from '../../components/grid'
+import GreyBackground from '../../components/Common/GreyBackground'
+import BlackBackground from '../../components/Common/BlackBackground'
+import TanBackground from '../../components/Common/TanBackground'
+
+import Layout from '../../components/layout'
+import Head from '../../components/Common/Head'
+import Hr from '../../components/Common/Hr'
+import CaseStudyHero from '../../components/Common/CaseStudyCards/CaseStudyHero'
+import { SectionTitle, BodyPrimary } from '../../components/Typography'
+import {
+  TextColumnsBlock,
+  shouldRender,
+  normalise,
+  normaliseAll,
+  BlockRow
+} from '../../components/Common/CaseStudyCards/CaseStudyBlocks'
+
+import CaseStudyPreview from '../../components/Common/CaseStudyCards/CaseStudyPreview'
+import Image from '../../components/Common/Image'
+
+const GreyBodyPrimary = styled(BodyPrimary)`
+  color: ${({ theme }) => theme.colors.secondaryText};
+`
+
+const Block3Col = styled(Col)`
+  padding-top: ${({ theme }) => theme.space[3]};
+`
+
+const Block4ImageCol = styled(Col)`
+  padding-top: ${({ theme }) => theme.space[4]};
+`
+
+const Block6ImageCol = styled(Col)`
+  padding-top: ${({ theme }) => theme.space[4]};
+`
+
+const Block7ImageCol = styled(Col)`
```

Comment:
You can use the new name as above and use `width={[]}` prop with a custom array to render or not depending on breakpoints

Reply:
This is an image column, I want it to display at all viewpoints but just give it some padding top on mobile down viewsa

Reply:
ok. If that's the reason we can leave that 👍